### ISSUE TITLE
Web page, not image?

### ISSUE THREAD

**Issue by glowkeeper** [chrislusf/seaweedfs#1591]
When I visit this `filer` url: 
```
http://localhost:8888/0x016EDD387B62F949284ED6579CEE175AD68B46D4/0x016EDD387B62F949284ED6579CEE175AD68B46D4.png
```
I'm hoping to see an image (a png). Instead, I get this page:

<img width="1440" alt="Screenshot 2020-11-01 at 19 48 08" src="https://user-images.githubusercontent.com/9264929/97812730-3fb03600-1c7b-11eb-8451-0d2a7e540d47.png">

What's the `filer` URL of the image itself (or how do I get to display it)?

**Comment by chrislusf** [comment]
this is fixed in https://github.com/chrislusf/seaweedfs/commit/9081ff8e65e60c6926e49c724fed20dfe1976d45

**Comment by glowkeeper** [comment]
Aha! Cool! Is that fix available in one of the available binaries?

**Comment by chrislusf** [comment]
not yet. building 2.08 as of now.

**Comment by glowkeeper** [comment]
Awesome - thanks...

**Comment by glowkeeper** [comment]
Will 2.08  include CORS support, too?

**Comment by glowkeeper** [comment]
I'm running 2.08 this morning, and it seems to be working - thanks!

Just FYI, When I first tried it, I got a `500` error when trying to load this URL, `http://localhost:8888/0x016EDD387B62F949284ED6579CEE175AD68B46D4/0x016EDD387B62F949284ED6579CEE175AD68B46D4.png`. The browser window told me `Volume 1 not found` (or something similar). However, I got it working by removing everything and then recreating the weed volume and redoing `filer.copy`. Not sure how I got there, but I might try and recreate the problem later. Right now, I have a deadline to meet, and now it's working, I'm going to get there ;)

Thanks for your help.
